x-pack/solutions/search/plugins/search_getting_started/public/components/footer/doc_links.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DocLinks } from '@kbn/doc-links';
+
+class ESDocLinks {
+  public visitSearchLabs: string = '';
+  public notebooksExamples: string = '';
+  public elasticsearchDocs: string = '';
+
+  constructor() {}
+
+  setDocLinks(newDocLinks: DocLinks) {
+    this.elasticsearchDocs = newDocLinks.elasticsearch.gettingStarted;
+    this.visitSearchLabs = newDocLinks.searchHomepage.visitSearchLabs;
+    this.notebooksExamples = newDocLinks.searchHomepage.notebooksExamples;
```

Comment:
Given that search homepage will be removing this content eventually, we should declare a new searchGettingStarted attribute in newDocLinks to gradually move references away from homepage. Don't remove searchHomepage attribute until after it's all done